ooni/utils/log.py

```diff
@@ -101,7 +102,7 @@ def emit(self, eventDict):
             self.write(text + "\n")
             self.flush()
 
-class MsecLogObserver(tw_log.FileLogObserver):
+class MsecLogObserver(LogLevelObserver):
```

Comment:
Not inheriting the `LogLevelObserver` will mean log levels are not respected and everything (even low priority logs) are written to a file.